ando/tools/generator/utils.py

```diff
@@ -0,0 +1,84 @@
+from os import path
+import pandas as pd
+import os
+import json
+
+
+def save_tsv(dataframe, path_to_save):
+    """
+
+    Parameters
+    ----------
+    dataframe: dataframe
+        dataframe to save in a TSV format
+    path_to_save: str
+        path to save the TSV file
+
+    Returns
+    -------
+
+    """
+    if path.exists(path_to_save):
+
+        df = pd.read_csv(os.path.join(path_to_save), sep='\t')
+        output = df.append(dataframe, sort=True)
+        output.to_csv(path_to_save, sep="\t", index=False)
+
+    else:
+        dataframe.to_csv(path_to_save, sep="\t", index=False)
+
+
+def save_json(new_dict, path_to_save):
+    """
+
+    Parameters
+    ----------
+    new_dict: dict
+        dict to save in a json format
+
+    path_to_save
+
+    Returns
+    -------
+
+    """
+    if path.exists(path_to_save):
+        with open(path_to_save) as json_file:
+            data_existing = json.load(json_file)
+            mergejson(data_existing, new_dict)
```

Comment:
after the merge the new dictionary is not written to disk. I think this part has been forgotten here maybe? If yes, then please also add a test that makes sure the merged data is really written to disk to prevent this mistake in the future.

Reply:
done